tests/instrumentation/benchmarks/base_perf_tests.py

```diff
@@ -0,0 +1,37 @@
+import mock
+import pytest
+
+import elasticapm
+from tests.fixtures import elasticapm_client
+
+from .perf_util import go_someplace_else
+
+capture_span = elasticapm.capture_span if hasattr(elasticapm, 'capture_span') else elasticapm.trace
+
+
+@pytest.mark.parametrize('elasticapm_client', [{
+    'include_paths': ('tests.instrumentation.perf_tests', 'instrumentation.perf_tests', 'a.b', 'c.d.e', 'f.g.h.x'),
+    'exclude_paths': ('tests.instrumentation.django_tests', 'elasticapm')
+}], indirect=True)
+def test_stacktrace_performance(elasticapm_client, benchmark):
+    with mock.patch('elasticapm.traces.TransactionsStore.should_collect') as should_collect:
```

Comment:
question you would never expect from me: why do you mock this?

Reply:
we're disabling the collection process here, as it might run in the background and suddenly collect transactions, which might influence the actual benchmark. There's other ways to do that, but this one's the easiest.